setup.py

```diff
@@ -5,6 +5,7 @@
 maintainer = 'Thomas Buchberger'
 
 tests_require = [
+    'ftw.file [bumblebee]',
```

Comment:
Since `ftw.blumblebee` is private, this makes it impossible to develop on this package for non-4teamwork developers, which defies the open source idea of this package.

`ftw.bumblebee` is private because it integrates the Bumblebee service, which is private too.
We should either make at least `ftw.bumblebee` public or move all integration-code into private integration repositories.